tests/utils/test_config.py

```
@@ -0,0 +1,120 @@
+            'GOOGLE_APPLICATION_CREDENTIALS': str(cred_file),
+            'GOOGLE_CLOUD_PROJECT': 'test-project',
+        }):
+            # Reload config module to pick up new env vars
+            import importlib
+            from pod_tenuki.utils import config
+            importlib.reload(config)
+
+            # Should not raise any exception
+            assert config.validate_config() is True
+
+    def test_validate_config_missing_auphonic_key(self, monkeypatch, tmp_path):
```

Ruff (0.14.5)

Unused method argument: `monkeypatch`

(ARG002)